### DISCUSSION TITLE
Best way to nuke the database?

### DISCUSSION THREAD

**Discussion by streetcap** [immich-app/immich#4932] [Q&A] [answered]
Hello,

I (almost) had my immich working perfectly a while back.  My biggest problem was that I had a lot of Synology generated thumbnail images that were included in the library index.  So I put in an Exclusion Pattern in the external library Scan Settings, forced a rescan of the index, then selected the option to remove any offline files.  That didn't seem to work correctly.  Without being able to get it just right, I ended up removing the external libraries (using the the Account Settings menu), selected remove offline files, then rebuilt the external libraries.  Still didn't work correct.  Repeated variations of this process a couple times.  No luck.

Anyways, Immich has turned into a hydra.  Every step I take to fix my library issues makes my library issues worse while also making immich run slower and slower to the point it is unusable (even if I pause all the background jobs).  

Unfortunately, the "Repair" function stopped working several versions ago and waiting for a release fix makes me feel like I would just be building on top of a messed up database anyways.  Fresh is always best ;)

Does anyone know the best way to clear out the database for the external images (preferably without impacting the regular immich default library) so I can REALLY rebuild this library from scratch? Or do a fresh install without losing all of my settings, user accounts, etc.?  

Thanks!

**Comment by alextran1502** [comment, marked as answer]
THis guide will help you https://immich.app/docs/FAQ#how-can-i-purge-data-from-immich

**Reply by streetcap** [reply]
Thank you for the help.  Do you know if I run that command to purge the metadata, will it also purge all of my settings, user accounts, etc.?

**Reply by alextran1502** [reply]
Yes it will

**Reply by streetcap** [reply]
Thank you.
